frontend/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Why do we need this file, which seems to only contain a comment?

Reply:
I think it's so that we can add type definitions for environment variables. See also [here](https://vitejs.dev/guide/env-and-mode#intellisense-for-typescript)